strings/round_start_sounds.txt

```diff
@@ -1,4 +1,7 @@
 sound/ambience/title1.ogg
 sound/ambience/title2.ogg
-sound/ambience/title3.ogg
 sound/ambience/clown.ogg
+config/title_music/bepoprush.ogg
```

Comment:
again, please put these in /haven/config/title_music instead.

Reply:
pointless change, tgs is a thing